### DISCUSSION TITLE
Selected View Sending to third party.

### DISCUSSION THREAD

**Discussion by vickykr26941** [nroduit/Weasis#598] [Q&A]
Hi, 
There is an option in weasis to capture the current view and save to local machine.
![image](https://github.com/user-attachments/assets/b1bdce1c-c906-4dec-84f0-985ff6273922)

Is there any option to send the current/selected  view with accessionNumber & studyInstanceUId over an third party api ?

**Comment by nroduit** [comment]
No, it's not possible, because there's no defined standardization for such an API.
